CMakeLists.txt

```diff
@@ -54,3 +54,8 @@ if (SLAC_INSTALL)
         EXPORT slac-targets
     )
 endif()
+
+if(BUILD_TESTING)
+    include(CTest)
```

Comment:
```suggestion
    include(CTest)
    enable_testing()
```

Reply:
`enable_testing ()` shouldn't be required if the option name is `BUILD_TESTING`.

Reply:
Wasn't sure about the case we are running the tests as dependency, but also then it shouldn't be required because `BUILD_TESTING` is set than, too.

https://cmake.org/cmake/help/latest/command/enable_testing.html